ui/components/Wallet/WalletActivityDetails.tsx

```diff
@@ -17,6 +17,19 @@ import { getBlockExplorerURL } from "../../utils/networks"
 function DetailRowItem(props: ActivityDetail): ReactElement {
   const { assetIconUrl, label, value } = props
 
+  function truncateValue(value: string) {
```

Comment:
What typo does this fix?

Reply:
resolves only the typo in messages.json 

the rest of the code is from a commit which is present here https://github.com/PelagusWallet/pelagus-extension/pull/94